src/uct/ib/mlx5/dc/dc_mlx5.c

```diff
@@ -901,7 +903,7 @@ uct_dc_mlx5_iface_dcis_create(uct_dc_mlx5_iface_t *iface,
 
     ucs_array_length(&iface->tx.dcis) = 0;
 
-    status = uct_dc_mlx5_iface_create_dci(iface, 0, 0);
+    status = uct_dc_mlx5_iface_create_dci(iface, 0, 0, 1);
```

Comment:
why just one channel and not `self->tx.num_dci_channels`?

Reply:
It doesn't matter as this dci is only created to query `bb_max` and then destroyed (lines 913-914)

Reply:
ic, probably need to change the name of this func, as it is quite confusing currently